### ISSUE TITLE
Bug: Some type definitions still has dependency on big-integer lib in 1.4.0 and 1.4.0-1

### ISSUE THREAD

**Issue by kevinava** [ip-num/ip-num#81]
Hi,
There seems to be some lingering dependencies for big-integer library in type definitions.
Have tried to re-install all node modules, have also checked that I have the latest release 1.4.0 and 1.4.0-1.
Errors only goes away when installing big-integer lib manually.

`Error: node_modules/ip-num/AbstractIPNum.d.ts:1:25 - error TS2307: Cannot find module 'big-integer' or its corresponding type declarations.
1 import * as bigInt from "big-integer";
                          ~~~~~~~~~~~~~
Error: node_modules/ip-num/AbstractIpRange.d.ts:1:25 - error TS2307: Cannot find module 'big-integer' or its corresponding type declarations.
1 import * as bigInt from "big-integer";
                          ~~~~~~~~~~~~~

Error: node_modules/ip-num/IPv4.d.ts:3:25 - error TS2307: Cannot find module 'big-integer' or its corresponding type declarations.
3 import * as bigInt from "big-integer";
                         ~~~~~~~~~~~~~

Error: node_modules/ip-num/IPv4CidrRange.d.ts:3:25 - error TS2307: Cannot find module 'big-integer' or its corresponding type declarations.
3 import * as bigInt from "big-integer";
                          ~~~~~~~~~~~~~

Error: node_modules/ip-num/IPv6.d.ts:3:25 - error TS2307: Cannot find module 'big-integer/BigInteger' or its corresponding type declarations.
3 import * as bigInt from "big-integer/BigInteger";
                          ~~~~~~~~~~~~~~~~~~~~~~~~

Error: node_modules/ip-num/IPv6CidrRange.d.ts:3:25 - error TS2307: Cannot find module 'big-integer' or its corresponding type declarations.
3 import * as bigInt from "big-integer";
                          ~~~~~~~~~~~~~

Error: node_modules/ip-num/SubnetMask.d.ts:2:25 - error[90m TS2307: Cannot find module 'big-integer' or its corresponding type declarations.
2 import * as bigInt from "big-integer";
                          ~~~~~~~~~~~~~

Error: node_modules/ip-num/interface/IPNumber.d.ts:1:28 - error TS2307: Cannot find module 'big-integer' or its corresponding type declarations.

1 import { BigInteger } from "big-integer";
                             ~~~~~~~~~~~~~

Error: node_modules/ip-num/interface/IPRange.d.ts:2:25 - error TS2307: Cannot find module 'big-integer' or its corresponding type declarations.
2 import bigInt = require("big-integer");                          ~~~~~~~~~~~~~
`

Cheers,
Kevin

**Comment by dadepo** [comment]
This is strange. In your issue I see for example:

> `Error: node_modules/ip-num/AbstractIPNum.d.ts:1:25 - error TS2307: Cannot find module 'big-integer' or its corresponding type declarations.
1 import * as bigInt from "big-integer";

But looking at the tagged source from which the v1.4.0 was made [here](https://github.com/ip-num/ip-num/blob/v1.4.0/src/IPNumber.ts), there is no occurrence of big-integer.

Can you try cleaning out your `node_module` folder?

**Comment by kevinava** [comment]
Hi,
I removed node_modules and and ran yarn install and got same result. 
So I started digging a little and found out the package on yarn is not updated, its still at 1.3.3.
Ref. [https://yarnpkg.com/package/ip-num](url) 

Note! package.json and yarn.lock says ip-num version 1.4.0
Ref. package.json
`"ip-num": "^1.4.0",`

Ref. yarn.lock 
```
ip-num@^1.4.0:
  version "1.4.0"
  resolved "https://registry.yarnpkg.com/ip-num/-/ip-num-1.4.0.tgz#2626eb39e908c4db01ca522c7a158896a8a43b08"
  integrity sha512-MP+gq4uBvrvm+G7EwP14GcJeFK49/p6sZrNOarMUoExLRodULJQM8mnkb/SbT1YKxRsZfh8rgwei2pUJIa35jA==
```

So clearly not a bug in lib, but a missing update of yarnpkg as data on yarn is consistent with version 1.3.3 with  big-integer lib requirement.

Cheers,
Kevin
